app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -52,34 +52,20 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             app:shadow_radius="8"
-            app:srcCompat="@{@drawable/ic_add_profile}"/>
+            app:srcCompat="@{@drawable/ic_default_avatar}"/>
```

Comment:
Wait, why is this changing? It seems unrelated to your PR.

Reply:
I was trying to resolve the merge conflict, however I've now reverted the changes back to original. PTAl